processing/src/main/java/io/druid/query/aggregation/hyperloglog/HyperLogLogCollector.java

```diff
@@ -640,7 +640,7 @@ private short addNibbleRegister(short bucket, byte positionOf1)
    * @param offsetDiff The difference in offset between the byteToAdd and the current HyperLogLogCollector
    * @param byteToAdd  The byte to merge into the current HyperLogLogCollector
    */
-  private static int mergeAndStoreByteRegister(
+  private static short mergeAndStoreByteRegister(
```

Comment:
I would prefer we leave the method signature unchanged and do an explicit cast to short whenever we use it, just to make sure, but that's just me being OC